<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="BoxProceso">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(82, 82, 82)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 18pt &quot;Myanmar Text&quot;;
color: rgb(241, 208, 22 )</string>
     </property>
     <property name="text">
      <string>Procesos</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>521</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGraphicsView" name="graphicsView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_8">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_10">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_14">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_15">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_16">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_17">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_18">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_19">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_20">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>5</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>6</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>6</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>75</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>75</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>75</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>75</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>76</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>76</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>75</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>75</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>76</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(241, 208, 22 );
color: white;
font-weight: bold;        
font-size: 18px;
border-radius: 15px;
padding: 10px;  </string>
     </property>
     <property name="text">
      <string>INICIAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BoxProductor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(82, 82, 82)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 18pt &quot;Myanmar Text&quot;;
color: rgb(241, 208, 22)</string>
     </property>
     <property name="text">
      <string>Productor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Tiempo restante:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>No. Productos: </string>
     </property>
    </widget>
    <widget class="QLabel" name="imageProductor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/productor.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_StatusProductor">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_StatusProductor_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_ProductosProductor">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="BoxProducto">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(82, 82, 82)</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 18pt &quot;Myanmar Text&quot;;
color: rgb(241, 208, 22 )</string>
     </property>
     <property name="text">
      <string>Consumidor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Tiempo restante:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Tiempo restante:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageConsumidor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/consumidor.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_StatusConsumer">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_TiempoConsumidor">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_TiempoConsumidor_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(241, 208, 22 );
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
